internal/exec/terraform_outputs.go

```
@@ -221,6 +221,8 @@ func execTerraformOutput(
 		)
 
 		// Terraform workspace
+	  backendType, ok := sections[cfg.BackendTypeSectionName].(string)
```

golangci-lint (1.64.8)

[error] File is not properly formatted

(gofumpt)